objects_test.go

```diff
@@ -49,7 +56,7 @@ func (s *ObjectsSuite) TestNewCommit(c *C) {
 	c.Assert(commit.Author.Name, Equals, "Máximo Cuadros")
 	c.Assert(commit.Author.When.Format(time.RFC3339), Equals, "2015-03-31T13:47:14+02:00")
 	c.Assert(commit.Committer.Email, Equals, "mcuadros@gmail.com")
-	c.Assert(commit.Message, Equals, "Merge pull request #1 from dripolles/feature\n\nCreating changelog\n")
+	c.Assert(commit.Message, Equals, "Merge pull request #1 from dripolles/feature\n\nCreating changelog")
```

Comment:
That's right. Before this PR, it was normalized during Decode. This PR removes the normalization step so we preserve the input as it is.
Otherwise, Encode/Decode would not be idempotent.